prospector/tools/ruff/__init__.py

```diff
@@ -11,6 +12,8 @@
 if TYPE_CHECKING:
     from prospector.config import ProspectorConfig
 
+_IGNORE_RE = re.compile(r"#\s*noqa:([^#]*[^# ])(?:\s*#.*)?$", re.IGNORECASE)
+
```

Comment:
Might want to add this to the ToolBase class for clarity ?

Reply:
+1 :-)

Reply:
Updated :-)

Reply:
The change is nice but I meant add the regex as an attribute in the ToolBase class to specify the regex for a tool noqa/disable explicitly (make it part of the ToolBase's API instead of creating a constant for each tool).

Reply:
Like the new commit?
I'm nor relay convinced, but it does the job :-)
(I should update the tests!)

Reply:
Yeap ! My reasoning is that the regex is strongly linked to a particular tool, but I don't mind if you revert, it's a nit.

Reply:
Yes, but how we use the regexp can also be linked to the tool :-)